src/user/commands/get-user-info.command.ts

```diff
@@ -0,0 +1,31 @@
+import { Injectable } from '@nestjs/common';
+import { PrismaService } from 'src/database/prisma.service';
+import { UserInfoDTO } from '../dto/user-info.dto';
+
+@Injectable()
+export class GetUserInfo {
+  constructor(private prisma: PrismaService) {}
+
+  async execute(UserId: number): Promise<UserInfoDTO> {
+    const user = await this.prisma.user.findFirst({
+      where: { id: UserId },
+    });
+
+    const student = await this.prisma.student.findFirst({
+      where: { user: { id: user.id } },
+    });
```

Comment:
Isso pode ser unido para fazer uma só consulta no banco:

```suggestion
    const user = await this.prisma.user.findFirst({
      where: { id: UserId },
      include: { student: true },
    });
```